src/status_im/contexts/shell/activity_center/view.cljs

```diff
@@ -36,23 +36,23 @@
     ;; AC after pressing the bell icon can be high.
```

Comment:
@ulisesmac, since are doing some changes in this function, I think we can remove this entire comment because it refers to a call way down below to `rn/delay-render`. But I'd say the comment is redundant because the function `rn/delay-render` is pretty clear already.

Reply:
Removed :+1: